src/core/server/config/deprecation/core_deprecations.ts

```diff
@@ -103,6 +103,17 @@ const mapManifestServiceUrlDeprecation: ConfigDeprecation = (settings, fromPath,
   return settings;
 };
 
+const requestLoggingEventDeprecation: ConfigDeprecation = (settings, fromPath, log) => {
+  if (has(settings, 'logging.events.request') || has(settings, 'logging.events.response')) {
+    log(
+      '"logging.events.request" and "logging.events.response" have been deprecated and will be removed ' +
+        'in 8.0. To access request/response data moving forward, please enable debug logs for the ' +
+        '"http.server.Kibana.response" context in your logging configuration.'
+    );
+  }
+  return settings;
+};
```

Comment:
The migration scenario for these is a little more complicated, so I created a custom deprecation here rather than using `unusedFromRoot` or `rename`. Happy to do this differently if anyone has strong feelings on it.

Reply:
A custom deprecation makes sense and has the added benefit of conveying info on how to handle the config change needed 😄